### DISCUSSION TITLE
[Feature Request] Ability to set a specific pickaxe requirement to mining spawners added to silk touch mining spawners requirement.

### DISCUSSION THREAD

**Discussion by AZFire1** [Zrips/CMI#4640] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Add a pickaxe requirement for mining spawners. For example, you have a list of available pickaxes used to mine spawners and you allow mining spawners with 2 pickaxes which are in the list. For example, diamond, and iron pickaxe. This requirement would be added onto the requirement of silk touch. 
### Basic explanation as to why this feature is needed and/or useful to have: ###
As I said above, another requirement or step to mine spawners (like silk touch). This requirement will allow you to set only specific pickaxes to be allowed to be used to mine spawners. Again, this would be added onto the silk touch requirement.

**Comment by CloudeLecaw** [comment]
Why you dont simply define a pickaxe with silktouch level 2 and define in cmi that silktouch 2 is needed to be able to mine the spawner?
Then give silktouch level 2 to you special pickaxe :)
This is possisble allready with cmi :)

U need to enable it and give the rigth permission and then simply set
```
    # Minimal silktouch level required to get spawner back
    SilkTouchLevel: 2
```

**Reply by 7hatGuyAJ** [reply]
Because that's not in the base vanilla game and not what he asked for, it would require a lot of setup and would spoil the game. I feel like having the ability to require tools based on material as well as enchants would be a huge feature that many would love. silk touch it easy to obtain so being able to have them also require a specific tool would not only slow progression it would also make it more fun.
